routes/api-routes.js

```diff
@@ -0,0 +1,16 @@
+// Dependencies
+var db = require("../models")
+
+//API routes
+module.exports = function(app){
+  //Get the route for profile information for one person
+  app.get("/api/members/:id", function(req, res){
+		db.Members.findOne({
+			where: {
+				id: req.params.id
+			},
+		}).then(function(dbMembers){
+			res.json(dbMembers);
+		});
+	});
+};
```

Comment:
Is `api-routes.js` being used anymore? I have been using `html-routes.js` and `listings/members-api-routes.js`

Reply:
Going to remove this...was using this for some testing on my device